src/main/java/org/sagebionetworks/bridge/exporter/record/BridgeExporterRecordProcessor.java

```diff
@@ -0,0 +1,177 @@
+
+    /** Config, used to get attributes for loop control and time zone. */
+    @Autowired
+    public final void setConfig(Config config) {
+        this.delayMillis = config.getInt(CONFIG_KEY_RECORD_LOOP_DELAY_MILLIS);
+        this.progressReportPeriod = config.getInt(CONFIG_KEY_RECORD_LOOP_PROGRESS_REPORT_PERIOD);
+        this.timeZone = DateTimeZone.forID(config.get(CONFIG_KEY_TIME_ZONE_NAME));
+    }
+
+    /** DDB Health Data Record table, used for querying full records for the list of record IDs. */
+    @Resource(name = "ddbRecordTable")
+    public final void setDdbRecordTable(Table ddbRecordTable) {
+        this.ddbRecordTable = ddbRecordTable;
+    }
+
+    /** File helper, used for creating and cleaning up the temp dir used to store the request's temporary files. */
+    @Autowired
+    public final void setFileHelper(FileHelper fileHelper) {
+        this.fileHelper = fileHelper;
+    }
+
+    /** Metrics helper, used for basic metrics operations. */
+    @Autowired
+    public final void setMetricsHelper(MetricsHelper metricsHelper) {
+        this.metricsHelper = metricsHelper;
+    }
+
+    /**
+     * Record filter helper, used to determine which records to filter out, due to request filters or sharing filters.
+     */
+    @Autowired
+    public final void setRecordFilterHelper(RecordFilterHelper recordFilterHelper) {
+        this.recordFilterHelper = recordFilterHelper;
+    }
+
+    /** Record ID source factor. Given the request, this generates the list of record IDs to be exported. */
+    @Autowired
+    public final void setRecordIdSourceFactory(RecordIdSourceFactory recordIdSourceFactory) {
+        this.recordIdSourceFactory = recordIdSourceFactory;
+    }
+
+    /**
+     * Worker manager, which takes export tasks and subtasks generated by this class and manages the workers and
+     * handlers to perform those tasks, and eventually uploads the export results to Synapse.
+     */
+    @Autowired
+    public final void setWorkerManager(ExportWorkerManager workerManager) {
+        this.workerManager = workerManager;
+    }
+
+    /**
+     * Main entry point into Bridge-EX. This process all records for the given request.
+     *
+     * @param request
+     *         request to process records for
+     * @throws IOException
+     *         if we fail to get the record IDs from the record ID factory
+     */
+    public void processRecordsForRequest(BridgeExporterRequest request) throws IOException {
+        LocalDate requestDate = request.getDate();
+        String requestTag = request.getTag();
+        LOG.info("Received request with date=" + requestDate + ", tag=" + requestTag);
+
+        // make task
+        Metrics metrics = new Metrics();
+        File tmpDir = fileHelper.createTempDir();
+        ExportTask task = new ExportTask.Builder().withExporterDate(LocalDate.now(timeZone)).withMetrics(metrics)
+                .withRequest(request).withTmpDir(tmpDir).build();
```

Comment:
Why time zone and not UTC? It doesn't seem like the request is effected by the time zone, since it's not specific to the request, it's configured.